megamek/src/megamek/common/icons/AbstractIcon.java

```diff
@@ -215,13 +215,29 @@ public void parseNodes(final NodeList nl) {
         }
     }
 
-    protected void parseNode(final Node wn) {
-        switch (wn.getNodeName()) {
+    /**
+     * Parses a given node and performs actions based on the node name.
+     *
+     * @param workingNode The node to parse
+     */
+    protected void parseNode(final Node workingNode) {
+        switch (workingNode.getNodeName()) {
             case "category":
-                setCategory(MMXMLUtility.unEscape(wn.getTextContent().trim()));
+                // setCategory(MMXMLUtility.unEscape(workingNode.getTextContent().trim()));
+
+                //start <50.01 compatibility handlers, the above-commented code can be uncommented
```

Comment:
The comments I've been using (you don't need to change this) has been 

// Remove Milestone post 0.49.19

And once the next Milestone is declared, going through and removing all said bridge code.